Source.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+constexpr int two{ 2 }, four{ 4 };// const-values
+
+ 
+int main()
+{
+	float a{}, b{}, c{};// 
+   //accept data from user
+	std::cout << "Enter A: "; std::cin >> a;
+	std::cout << "Enter B: "; std::cin >> b;
+	std::cout << "Enter C: "; std::cin >> c;
+  
+	float* res = nullptr;//result`s pointer
+	float discriminant = std::move(pow(b, two) - static_cast<double> (four )* a * c);//calculation of discriminant

```

Comment:
```suggestion
	float discriminant = pow(b, two) - static_cast<double> (four )* a * c;//calculation of discriminant
```
перемещение для примитивных типов не работает, это обычное копирование
сам смысл move это семантика которая говорит о том как сделать лучше копирование